packages/client/package.json

```diff
@@ -11,6 +11,7 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-firebase-hooks": "^5.1.1",
+    "react-icons": "^4.7.1",
```

Comment:
Isn't there already an icon in the assets for this? Else I'd talk to the designer about that.

Reply:
the Icons that I wanted to use wasn't there